Formula/ffmpeg.rb

```diff
@@ -110,11 +125,13 @@ def install
     ]
 
     if build.with? "openh264"
-      resource("openh264-1.5.0").stage do
-        system "make", "install-shared", "PREFIX=#{libexec}/openh264-1.5.0"
-        chmod 0444, libexec/"openh264-1.5.0/lib/libopenh264.dylib"
+      if build.stable?
+        resource("openh264-1.5.0").stage do
+          system "make", "install-shared", "PREFIX=#{libexec}/openh264-1.5.0"
+          chmod 0444, libexec/"openh264-1.5.0/lib/libopenh264.dylib"
```

Comment:
I honestly do not know, this is a vendor that I believe @ilovezfs did, but I just left it intact to avoid breaking something I did not understand. I can look into it, but as I remember it was pulled from the `openh264` formula's install block.